kwamp-client-core/src/main/kotlin/com/laurencegarmstrong/kwamp/client/core/Client.kt

```diff
@@ -119,6 +122,7 @@ class ClientImpl(
         connection.withNextMessage { message: Welcome ->
             log.info("Session established. ID: ${message.session}")
             sessionId = message.session
+
```

Comment:
To be consistent with the rest of the code layout, can we remove this newline? Is there another reason it's here?